common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java

```diff
@@ -0,0 +1,215 @@
+            pageDestination = currentPage;
+        } else if (pageDestination != currentPage && quickJumping) {
+            jumpToDestination();
+        } else {
+            quickJumping = false;
+        }
+    }
+
+    @SubscribeEvent
+    public void onContainerClose(ContainerCloseEvent.Post e) {
+        isBankScreen = false;
+        currentPage = 1;
+        pageDestination = 1;
+    }
+
+    @SubscribeEvent
+    public void onSlotClicked(ContainerClickEvent e) {
+        if (!isBankScreen) return;
+
+        int slotIndex = e.getSlotNum();
+
+        if (BUTTON_SLOTS.contains(slotIndex)) {
+            int buttonIndex = BUTTON_SLOTS.indexOf(slotIndex);
+            pageDestination = customJumpDestinations.get(buttonIndex);
+            e.setCanceled(true);
+            jumpToDestination();
+        }
+    }
+
+    @SubscribeEvent
+    public void onRenderSlot(SlotRenderEvent.Pre e) {
+        if (!(e.getScreen() instanceof AbstractContainerScreen<?> screen)) return;
+        if (!isBankScreen) return;
+
+        if (BUTTON_SLOTS.contains(e.getSlot().index)) {
+            ItemStack jumpButton = new ItemStack(Items.DIAMOND_AXE);
+            jumpButton.setDamageValue(92);
+
+            CompoundTag jumpTag = jumpButton.getOrCreateTag();
+            jumpTag.putInt("HideFlags", 6);
+            jumpTag.putBoolean("Unbreakable", true);
+            jumpButton.setTag(jumpTag);
+
+            int buttonIndex = BUTTON_SLOTS.indexOf(e.getSlot().index);
+            int buttonDestination = customJumpDestinations.get(buttonIndex);
+
+            jumpButton.setCount(buttonDestination);
+
+            jumpButton.setHoverName(Component.literal(ChatFormatting.GRAY + "Jump to Page " + buttonDestination));
+
+            e.getSlot().set(jumpButton);
+        }
+
+        if (e.getSlot().index == NEXT_PAGE_SLOT && Models.Container.isLastBankPage(screen)) {
+            lastPage = currentPage;
+        }
+    }
+
+    private void jumpToDestination() {
+        quickJumping = true;
+
+        if (currentPage == pageDestination) return;
+
+        if (currentPage + 1 == pageDestination && lastPage != currentPage) {
+            clickNextPage();
+        } else if (currentPage - 1 == pageDestination) {
+            clickPreviousPage();
+        } else {
+            int closest = QUICK_JUMP_DESTINATIONS.get(0);
+            int closestDistance = Math.abs(closest - pageDestination);
+            int currentDistance = Math.abs(currentPage - pageDestination);
+
+            for (int jumpDestination : QUICK_JUMP_DESTINATIONS) {
+                int jumpDistance = Math.abs(jumpDestination - pageDestination);
+
+                if (jumpDistance < closestDistance) {
+                    closest = jumpDestination;
+                    closestDistance = jumpDistance;
+                }
+            }
+
+            if (closestDistance < currentDistance) {
+                clickJumpButton(QUICK_JUMP_DESTINATIONS.indexOf(closest));
```

Comment:
Can you extract this into a tryUsingJumpButtons(), that returns true if successful?